x-pack/plugins/reporting/export_types/png/server/create_job/index.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { cryptoFactory } from '../../../../server/lib/crypto';
+import { oncePerServer } from '../../../../server/lib/once_per_server';
+import { compatibilityShimFactory } from '../../../common/create_job/compatibility_shim';
```

Comment:
I think you can rip out the compatibility shim stuff, and keep that only in the pdf section.  This is to support backward compatibility for reporting urls that were generated prior to some refactoring.  Since png is new, we don't have to support any old style urls.

That sound about right, @kobelb?

Reply:
Yup, that's right.